### PR TITLE
Added Your responsibility

### PR DESCRIPTION
**Pull request by frsfx**
@diego- Styling needs to be done for bullet list but I wanted you to review the content and provide comments.  Probably will need feedback on the docker container and maybe some content on time deployments.  I also created images that I would like to put in if possible.

### CODE REVIEW

**Comment by diego-** [comment]
@dlapiduz

File: content/intro/overview/what-is-cloudgov.md [hunk 1]
```diff
@@ -22,3 +22,32 @@ A byproduct of cloud.gov’s creation is [compliance-masonry](https://github.com
...
 - What is needed for "Government-as-a-Platform" services, including PaaS-related container and clustering technologies, and whether to incorporate these into our toolbox.
+
+### What is your responsibility
+cloud.gov is a FedRAMP compliant Platform as a Service (Paas).  What this means is that agencies can deploy their application on a FedRAMP compliant platform and just be responsible app code.  The way it works is cloud.gov’s compliance is encapsulated within a boundary.  All components within that boundary are cloud.gov’s responsibility in terms of maintaining and operating the platform.  The specific items that cloud.gov is responsible for are the following:
+- Infrastructure
+- Supported Buildpacks
+- OS
+- Anti-Malware
+- Network Security 
+- Continuous Monitoring
```
Comment (dlapiduz) [inline content/intro/overview/what-is-cloudgov.md]: For all of these we should say "Platform..." so "Platform Cotinuous monitoring", "Platform versioning" etc

**Comment by dlapiduz** [comment]
Thanks @diego- :)

**Comment by dlapiduz** [comment]
Nice graph!

**Comment by frsfx** [comment]
@brittag when you have a moment please review and let me know if you have any feedback.

**Comment by afeld** [comment]
@frsfx Thanks so much for doing this! Since this is an Introduction page, I think we can simplify a lot of the language. Will send a pull request on top of this one on Tuesday!

**Comment by brittag** [comment]
I adapted this content and diagram into the merged PR at https://github.com/18F/cg-docs/pull/309, so I'm going to close this one. Thanks again @frsfx for writing this!

**Pull request closed without merging**